test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +35,36 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNull();
+        
+        // only one object
+        assertAreNull((Object) null);
+        assertNotNull(1);
+        assertNotNull("");
+        assertNotNull("abc");
+        
+        // some identical objects
+        assertAreNull(null, null);
+        assertNotNull(1, 1);
+        assertNotNull("abc", "abc");
+        
+        // different ordering of null
+        assertAreNull(null, 1, 2);
+        assertAreNull(1, null, 2);
+        assertAreNull(1, 2, null);
+    }
+
+    private void assertAreNull(Object... objects) {
```

Comment:
Extracting the assert function is not really necessary here since this method is quite simple (with only one line). Thus, you can add it directly into your test cases.